CMakeLists.txt

```diff
@@ -22,16 +22,6 @@ cmake_minimum_required(VERSION 2.8)
 # CMake's Fortran tests
 SET(CMAKE_Fortran_COMPILER_WORKS 1)
 
-
-set(CMAKE_Fortran_PREPROCESS_SOURCE
-    "<CMAKE_Fortran_COMPILER> -cpp <DEFINES> <INCLUDES> <FLAGS> -E <SOURCE> -o <PREPROCESSED_SOURCE>")
-
-# If we are not building as a part of LLVM, build Flang as an
-# standalone project, using LLVM as an external library:
-if( CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR )
-  project(Flang)
-endif()
```

Comment:
Why is this removed? When this is removed, the 2 lines below will not work

Reply:
IMO, last 3 PRs should be reversed

Reply:
See diff here: https://github.com/flang-compiler/flang/pull/288/files#diff-af3b638bc2a3e6c650974192a53c7291